src/Chance.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * Created by EddyJ on 6/14/16.
+ */
+public class Chance {
+    private String title;
+    private String description;
+
+    public Chance(String title, String description){
```

Comment:
This is a great way to do it.  Neither have to change so put them in a constructor.  Nice work